### PR TITLE
Fixed ESNZ location and Sweden/Finland border overlaps

### PR DESCRIPTION
**Pull request by felixholmertz**

### CODE REVIEW

**Review by NelisV — CHANGES_REQUESTED** [review]
The ones you changed look good, although some gaps could still be fixed: see the red lines
image

**Comment by felixholmertz** [comment]
> The ones you changed look good, although some gaps could still be fixed: see the red lines

Quite a lot more than those red lines that were off but it's fixed in cd2d60aff7c7d20b4c53115bf28d0cfad3c01af2

File: FIRBoundaries.dat [hunk 1]
```diff
@@ -3080,60 +3080,59 @@ EFIN|0|0|80|59.000000|19.086667|70.116667|31.583333|63.733333|27.000000
...
-68.066667|29.35
-67.8|29.683333
+68.866667|28.800000
+68.550000|28.450000
+68.200000|28.666667
+68.066667|29.350000
+67.800000|29.683333
 67.666667|30.016667
-67.516667|29.95
+67.516667|29.950000
```
Comment (NelisV) [inline FIRBoundaries.dat:3105]: Could you check these changes? in a few cases you're just adding zero's while we just added a PR to round them all correctly

Comment (felixholmertz) [reply FIRBoundaries.dat]: Fixed in c1764d049cf9be535b1aa8ea006c3ca360d65d91. Removed other commit

**Review by Adrian2k — APPROVED** [review]
I will approve the PR, however @NelisV will have to check for any remaining issues with the border as he is the QGIS expert here! 😄

**Review by NelisV — APPROVED** [review]
Thanks! Looks good now :)

**Pull request merged**